### DISCUSSION TITLE
Showly unavailable in some countrys?

### DISCUSSION THREAD

**Discussion by enzon19** [trakt/showly#597] [General]
I'm trying to download Showly in a brazilian account, but I receive and error from Google Play saying that it's unavaliable for my country.
![image](https://user-images.githubusercontent.com/89558747/171522250-5f33ab9b-d116-481b-b143-09b5219cf1cc.png)

**Comment by michaldrabik** [comment]
@enzon19 Yes Brasil is currently unavailable. Had some problems there. Working on bringing it back. 

Feel free to download the latest APK from Github Releases page though.

**Reply by enzon19** [reply]
Yes, I did this! Thanks!
